web/modules/custom/hoeringsportal_audit_log/src/Form/SettingsForm.php

```diff
@@ -203,13 +201,14 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
    * @param string $typeId
    *   typeId.
    *
-   * @return array<string, string>
+   * @return array<string, string>|bool
    *   Default values from config.
    */
-  private function getDefaultValues(string $definitionId, string $typeId) : array {
+  private function getDefaultValues(string $definitionId, string $typeId) : array|bool {
     $configTypes = $this->configHelper->getConfiguration('types');
     $defaultValues = [];
-    if (count($configTypes) > 0) {
+
+    if ($configTypes && is_array($configTypes) && count($configTypes) > 1) {
```

Comment:
Why must the count be greater than 1 (and not just 0)?

Reply:
I guess ` && count($configTypes) > 0` is already being checked in `$configTypes` so I'll just remove it.